### DISCUSSION TITLE
Swapping Languages for different characters?

### DISCUSSION THREAD

**Discussion by Scaramoucher** [erew123/alltalk_tts#320] [Q&A] [answered]
Is there a way to set different languages for different characters in SillyTavern when using XTTS? Manually switching the language over has the narrator share that language, rather than just affecting the specific character's speech. 

Just wondering if it's possible to have specific characters speak languages other than English without changing the narrator's (or a separate character's) language? 

I appreciate the time.

**Comment by erew123** [comment, marked as answer]
Hi @Scaramoucher 

That type of information is stored in the SillyTavern character map (a group of settings linked to the character card). Its not something I could change/alter if I wanted to. You would have to request that from the Dev's at SillyTavern.

Thanks
